pkgs/development/libraries/gtk/2.x.nix

```diff
@@ -21,11 +21,11 @@ in
 
 stdenv.mkDerivation rec {
   pname = "gtk+";
-  version = "2.24.32";
+  version = "2.24.33";
 
   src = fetchurl {
     url = "mirror://gnome/sources/gtk+/2.24/${pname}-${version}.tar.xz";
```

Comment:
```suggestion
    url = "mirror://gnome/sources/gtk+/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
```

Reply:
This is the last release so I would not really bother.